Code/Tools/Android/ProjectBuilder/AndroidManifest.xml

```diff
@@ -64,8 +64,8 @@
         <service android:name="com.android.vending.billing.InAppBillingService" />
 
         <!-- Services for downloading the Obb -->
-        <service android:name="com.amazon.lumberyard.io.obb.ObbDownloaderService" />
-        <receiver android:name="com.amazon.lumberyard.io.obb.ObbDownloaderAlarmReceiver" />
+        <service android:name="com.amazon.lumberyard.io.obb.ObbDownloaderService" android:exported="true" />
+        <receiver android:name="com.amazon.lumberyard.io.obb.ObbDownloaderAlarmReceiver" android:exported="true" />
 
```

Comment:
Why is this set to true?  We don't use OBBs.

Reply:
Not necessary.
Reverted.